test/features/steps/steps.py

```diff
@@ -0,0 +1,44 @@
+from behave import when
+from behaving.web.steps import *  # noqa: F401, F403
+from behaving.personas.steps import *  # noqa: F401, F403
+from behaving.web.steps.url import when_i_visit_url
+from behaving.mail.steps import *
+import random
+
+@when('I go to homepage')
```

Comment:
`@step`

Reply:
hard to say if it should be "when", or a "then" or a "and" per https://cucumber.io/docs/gherkin/reference/#steps

maybe a 'and' would be more suitable as it fit under a given. 
given persona 'x' logs in
and i go to the homepage
when the homepage has finished loading
then i should see that the logout button
but i should not see the login button